tests/bpf2c_tests/expected/tail_call_max_exceed_dll.c

```diff
@@ -7640,47 +7640,19 @@ static void
 _get_version(_Out_ bpf2c_version_t* version)
 {
     version->major = 0;
-    version->minor = 10;
+    version->minor = 11;
     version->revision = 0;
 }
 
 #pragma data_seg(push, "map_initial_values")
 static const char* _bind_tail_call_map_initial_string_table[] = {
-    "bind_test_callee0",
-    "bind_test_callee1",
-    "bind_test_callee2",
-    "bind_test_callee3",
-    "bind_test_callee4",
-    "bind_test_callee5",
-    "bind_test_callee6",
-    "bind_test_callee7",
-    "bind_test_callee8",
-    "bind_test_callee9",
-    "bind_test_callee10",
-    "bind_test_callee11",
-    "bind_test_callee12",
-    "bind_test_callee13",
-    "bind_test_callee14",
-    "bind_test_callee15",
-    "bind_test_callee16",
-    "bind_test_callee17",
-    "bind_test_callee18",
-    "bind_test_callee19",
-    "bind_test_callee20",
-    "bind_test_callee21",
-    "bind_test_callee22",
-    "bind_test_callee23",
-    "bind_test_callee24",
-    "bind_test_callee25",
-    "bind_test_callee26",
-    "bind_test_callee27",
-    "bind_test_callee28",
-    "bind_test_callee29",
-    "bind_test_callee30",
-    "bind_test_callee31",
-    "bind_test_callee32",
-    "bind_test_callee33",
-    "bind_test_callee34",
+    "bind_test_callee0",  "bind_test_callee1",  "bind_test_callee2",  "bind_test_callee3",  "bind_test_callee4",
```

Comment:
nit: I think this change is going to fail bpf2c test. For some reason `format-code.ps1` is changing this, but then it does not match with the actual output, causing bpf2c output to fail.

If the tests fail, you may want to revert this part of the change, and use `--no-verify` option in `git commit`.

Reply:
bpf2c tests did fail. You will need to revert this part of the change for tests to pass.